docs/settings.md

```diff
@@ -47,6 +47,40 @@ Content-Type: application/vnd.api+json
 }
 ```
 
+## OAuth clients usage
+
+### GET /settings/clients-usage
+
+This endpoint returns the number of user-connected OAuth clients, the limit set
+on the Cozy and if this limit has been reached or even exceeded.
+If there is no limit, the attribute won't be present in the response.
+
+#### Request
+
+```
+GET /settings/clients-usage HTTP/1.1
+Host: alice.example.com
+Accept: application/vnd.api+json
+Authorization: Bearer ...
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+```
+
+```json
+{
+  "limit": 3,
+  "count": 3,
+  "reached": true,
+  "exceeded": false
+}
+```
```

Comment:
I think the response should in the JSON-API format

Reply:
Same here, I followed the same pattern as the disk usage one which I believe does not use the JSON-API format.

Reply:
https://docs.cozy.io/en/cozy-stack/settings/#get-settingsdisk-usage : it is in JSON-API format